lambda/custom/index.js

```diff
@@ -120,13 +138,52 @@ function handleUserGuess(userGaveUp, handlerInput) {
   }
 
   // Check if we can exit the game session after GAME_LENGTH questions (zero-indexed)
+  
```

Comment:
Please audit all of the whitespace you added and ensure it is intended.

Reply:
Will do